src/utils/transaction.ts

```diff
@@ -43,43 +41,39 @@ import { ITransactionBlock } from '../types/transactionBlock';
 import { POLYGON_USDC_CONTRACT_ADDRESS } from '../constants/assetConstants';
 
 export const klimaDaoStaking = async (
-  amount: string,
+  routeToKlima?: BridgingQuote | null,
   receiverAddress?: string,
   sdk?: EtherspotSdk | null,
-): Promise<{ errorMessage?: string; result?: { transactions: ICrossChainActionTransaction[], returnAmount: BigNumber, provider?: ExchangeProviders } }> => {
+  flag?: Boolean,
+  amount?: string,
+): Promise<{ errorMessage?: string; result?: { transactions: ICrossChainActionTransaction[], provider?: string } }> => {
   if (!sdk) return { errorMessage: 'No sdk found' };
 
+  if (!routeToKlima) return { errorMessage: 'No routes found' }
```

Comment:
The error message should instruct the user what to do next, as mentioned above.

Reply:
Yeah so if the route is not found then it must because of low amount for cross chain from LiFi side for the pair to exchange or might be due to liquidation issues for the pair so there are a lot, a user can do here for eg change the amount or change the source token